### PR TITLE
+ added page action

### PR DESCRIPTION
**Pull request by lepirata**
I've added an icon in the address bar that let's you add/remove the page faster to the reading list.
Resolves issue #81

### CODE REVIEW

**Comment by whitecat** [comment]
Thank you for the commit. Can you make an issue for this so we can use it to keep track of changes?

File: src/_locales/de/messages.json [hunk 1]
```diff
@@ -35,6 +35,9 @@
   "context": {
     "message": "Im Kontextmenü anzeigen:"
   },
+  "pageActionOption": {
+    "message": "Weergeven in adresbalk:"
```
Comment (whitecat) [inline src/_locales/de/messages.json]: I would use `Im Adressleiste anzeigen,` but if you are German that is fine. The word `Weergeven` is dutch.

File: src/js/options.js [hunk 2]
```diff
@@ -42,10 +48,21 @@ document.addEventListener('DOMContentLoaded', () => {
       items.settings.theme = theme
       items.settings.animateItems = animateItems
       items.settings.addContextMenu = addContextMenu
+      items.settings.addPageAction = addPageAction
       items.settings.openNewTab = openNewTab
       chrome.storage.sync.set({
         settings: items.settings
       })
+
+      chrome.tabs.query({}, tabs => {
```
Comment (whitecat) [inline src/js/options.js]: Line 57-65 should be above line the line with the comment  `// Get updating the settings on the options page.`
It should also have a comment on what it is doing.

File: src/js/options.js [hunk 3]
```diff
@@ -13,17 +13,23 @@ document.addEventListener('DOMContentLoaded', () => {
...
       theme: 'light',
       addContextMenu: true,
+      addPageAction: true,
       animateItems: !isFirefox,
       openNewTab: false,
       viewAll: true
     }
   }
 
+  if (!isFirefox) {
```
Comment (whitecat) [inline src/js/options.js]: This is awesome! I really like this good job!

Add comment so we know why it is there in the future. something like:
`// Icon in address bar only for firefox does not work in chrome`

File: src/js/background.js [hunk 4]
```diff
@@ -206,9 +217,36 @@ chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
 })
 
 chrome.tabs.onActivated.addListener((tabId, windowId) => {
+  if (isFirefox) {
```
Comment (whitecat) [inline src/js/background.js]: I do not think this code block is necessary. I believe it is covered by the update code block.

**Comment by whitecat** [comment]
Thank you so much for your patience. You have been wonderful!

**Comment by whitecat** [comment]
@lepirata We should merge it this afternoon.

**Comment by alexpdraper** [comment]
This is really cool. Excellent work.

**Pull request merged**